src/regionprops.jl

```diff
@@ -202,12 +246,67 @@ function cropfloe(floesimg::BitMatrix, props::DataFrame, i::Int64)
     return prefloe
 end
 
+"""
+    cropfloe(floesimg, min_row, min_col, max_row, max_col, label)
+
+Crops the floe from `floesimg` with the label `label`, returning the region bounded by `min_row`, `min_col`, `max_row`, `max_col`, and converting to a BitMatrix.
+"""
+function cropfloe(floesimg::Matrix{I}, min_row::J, min_col::J, max_row::J, max_col::J, label::I)  where {I<:Integer, J<:Integer}
```

Comment:
Because `I` and `J` are the same type, they can be combined into one.
```suggestion
function cropfloe(floesimg::Matrix{T}, min_row::T, min_col::T, max_row::T, max_col::T, label::T)  where {T<:Integer}
```

Reply:
Not quite. They might be different integer types. The matrix values `I` have to be the same concrete type as the labels. But the column and row numbers might be a different concrete integer type. 

Concrete: You might have a `UInt8` for the labels, with maximum of 256 individual labels for 255 different floes and a background, but you'll probably need more than that for the row and column indices – we're likely in the `UInt16` range with up to 65563 pixels on a side.

Reply:
I see your point, but I am looking at this a bit more abstractly. For the purposes of multiple dispatch and argument types in function signatures, specially when using the same abstract type, the directive `where {I<:Integer, J<:Integer}` has the same effect as `where {T<:Integer}` and use `T` for both `I` and `J`. They do the same thing but the version in the suggestion is shorter.

Reply:
I don't think that's correct. They don't do the same thing, because the `{T<:Integer}` means `T` is going to be a single type, which is a subtype of `Integer`, and all the arguments are going to be of that type. So it will match to `a::Int8, b::Int8, c::Int8` but not `a::int8, b::Int16, c::Int32`.

I did a quick example here: https://colab.research.google.com/drive/1K35IImi2g8AF1KXfp59eckrnjyCCrEc2?usp=sharing